test/general/key.js

```diff
@@ -2553,6 +2647,79 @@ function versionSpecificTests() {
     }
   });
 
+  it('Sign and verify a key with user attribute - all users', async function () {
+    let publicKey = await openpgp.readKey({ armoredKey: keyWithImageData });
+    const privateKey = await openpgp.decryptKey({
+      privateKey: await openpgp.readKey({ armoredKey: priv_key_rsa }),
+      passphrase: 'hello world',
```

Comment:
Thanks! The linter still complains about the trailing comma here (and in the next test). You can also run it locally with `npm run lint` :blush:

Reply:
thanks, and sorry for the back and forth 😅

Reply:
No worries, and thank for following up on everything! :D